rad/src/script.rs

```diff
@@ -0,0 +1,144 @@
+use crate::error::*;
+use crate::operators::{operate, RadonOpCodes};
+use crate::types::RadonTypes;
+
+use log::error;
+use num_traits::FromPrimitive;
+use rmpv::{self, Value};
+use std::{error::Error, io::Cursor};
+use witnet_util::error::WitnetError;
+
+pub type RadonCall = (RadonOpCodes, Option<Vec<Value>>);
```

Comment:
I find `Option<Vec<...>>` a bit redundant since `Vec<...>` alone is able to express the same as `Option<...>`. Is there a reason that prevents us from using just `pub type RadonCall = (RadonOpCodes, Vec<Value>);`

Reply:
The reason is that you can't tell an empty vector from a filled one without using a guard :rofl:

Reply:
But, isn't the same situation with `Option`? You need to test if it filled or not right?

Reply:
But testing for length in a guard for every branch of the `match` is a pain in the ass.

Reply:
I mean, the `match` is explicit about whether each operator accepts arguments or not.